data/fixture/test/tasks.js

```diff
@@ -40,5 +40,15 @@ module.exports = [
 		url: 'nature.com/news',
 		timeout: 30000,
 		standard: 'Section508'
+	},
+	{
+		_id: new ObjectID('abc000000000000000000004'),
+		name: 'Z Integration Test',
```

Comment:
The `Z` is so that this entry will be after all the other entires, as they get alphabetically ordered.